arangod/Aql/IResearchViewExecutor.tpp

```diff
@@ -1105,18 +1132,18 @@ inline void commonReadPK(ColumnIterator& it, irs::doc_id_t docId,
 }
 
 template<typename ExecutionTraits>
-bool IResearchViewExecutor<ExecutionTraits>::readPK(
-    LocalDocumentId& documentId) {
+bool IResearchViewExecutor<ExecutionTraits>::readPK(LocalDocumentId& documentId,
```

Comment:
make it just anonymous function, it doesn't need this now

Reply:
is is already static